charts/astronomer/values.yaml

```diff
@@ -202,11 +202,11 @@ houston:
     # url for update service
     url: https://updates.astronomer.io/astronomer-runtime
 
-    # Default here is to run at 00:43 every night
+    # Default here is to run at 00:43 every night https://crontab.guru/#43_0_*_*_*
     schedule: "43 0 * * *"
-  # this option allows to override airflow releases config
+  
 
-  #
+  # This option allows to override airflow releases config
```

Comment:
This comment seems redundant. Do we think it's going to be helpful, given the context and the fact that the last three words are the exploded version of the variable name?